Makefile.am

```diff
@@ -56,6 +56,8 @@ check_PROGRAMS += \
 capn_test_SOURCES = \
 	tests/capn-test.cpp \
 	tests/capn-stream-test.cpp \
+	tests/example-test.cpp \
+	tests/example.capnp.c \
 	compiler/test.capnp.c \
 	compiler/schema-test.cpp \
 	compiler/schema.capnp.c
```

Comment:
Fixed. Sorry, first time with automake.
Output of `make dist; tar -tvf c-capnproto-0.2.tar.gz | sort -f6`:
```
drwxrwxr-x 1000/1000         0 2017-03-24 17:58 c-capnproto-0.2/tests/
-rw-rw-r-- 1000/1000      7752 2017-03-23 00:36 c-capnproto-0.2/tests/capn-stream-test.cpp
-rw-rw-r-- 1000/1000     14902 2017-03-23 00:36 c-capnproto-0.2/tests/capn-test.cpp
-rw-rw-r-- 1000/1000       178 2017-03-23 20:18 c-capnproto-0.2/tests/example.capnp
-rw-rw-r-- 1000/1000      1826 2017-03-23 20:18 c-capnproto-0.2/tests/example.capnp.c
-rw-rw-r-- 1000/1000      1179 2017-03-23 20:18 c-capnproto-0.2/tests/example.capnp.h
-rw-rw-r-- 1000/1000      1818 2017-03-24 02:31 c-capnproto-0.2/tests/example-test.cpp
```
I noticed the other pre-existing `.capnp` files weren't in `EXTRA_DIST` but when I tried adding them they broke `make dist` with this output:
```
 % make dist
 cd . && /bin/bash /home/alex/Code/stm32/modules/arm-gcc-toolchain/linux/c-capnproto/source/ac/missing automake-1.15 --foreign Makefile
 cd . && /bin/bash ./config.status Makefile depfiles
config.status: creating Makefile
config.status: executing depfiles commands
make  dist-gzip am__post_remove_distdir='@:'
make[1]: Entering directory '/home/alex/Code/stm32/modules/arm-gcc-toolchain/linux/c-capnproto/source'
gcc -g -O2    compiler/test.capnp.c   -o compiler/test.capnp
In file included from compiler/test.capnp.c:1:0:
compiler/test.capnp.h:4:21: fatal error: capnp_c.h: No such file or directory
compilation terminated.
<builtin>: recipe for target 'compiler/test.capnp' failed
make[1]: *** [compiler/test.capnp] Error 1
make[1]: Leaving directory '/home/alex/Code/stm32/modules/arm-gcc-toolchain/linux/c-capnproto/source'
Makefile:1485: recipe for target 'dist' failed
make: *** [dist] Error 2
```
Any ideas how to fix this?